controllers/servicefence_controller.go

```diff
@@ -376,17 +375,61 @@ func addDomainsWithHost(domains map[string]*lazyloadv1alpha1.Destinations, sf *l
 
 	for h, strategy := range sf.Spec.Host {
 
+		// handle namespace level host, like 'default/*'
+		if strings.HasSuffix(h, "/*") {
+			subHosts := strings.Split(h, "/")
+			if len(subHosts) != 2 {
+				log.Errorf("%s is invalid host, skip", h)
+				continue
+			}
+			nsSvcCache.RLock()
+			svcs := nsSvcCache.Data[subHosts[0]]
+			var allHost []string
+			for svc := range svcs {
+				subSvcs := strings.Split(svc, "/")
+				service := fmt.Sprintf("%s.%s.svc.cluster.local", subSvcs[1], subSvcs[0])
```

Comment:
more accurate var name

Reply:
use `xxxParts` instead `subxxx` and use `fullHost` instead `service`